src/configparser.cpp

```diff
@@ -102,7 +105,7 @@ bool ConfigParser::parse_file(const std::string& tmp_filename)
 		if (!line.empty() && line.back() == '\\') {
 			multi_line_buffer.append(line.substr(0, line.size()-1));
 		} else {
-			const std::string location = strprintf::fmt(_("%s line %u"), filename, linecounter);
+			const newsboat::Filepath location = strprintf::fmt(_("%s line %u"), filename, linecounter);
```

Comment:
Please revert this change, because `location` is not a path to a file — it's a part of the error message for the user.